File: C++/1040.cpp
#include <stdio.h>


 
int main() {
 float n1,n2,n3,n4,media1,nf,mediaf;
 
 scanf("%f %f %f %f",&n1,&n2,&n3,&n4);
   media1=(((n1*2)+(n2*3)+(n3*4)+n4)/10);
   printf("Media: %.1f\n",media1);
   if (media1>=7.0) printf ("Aluno aprovado.\n");
   if (media1<5.0) printf ("Aluno reprovado.\n");
   if (media1>=5.0 && media1<7.0){
   	printf("Aluno em exame.\n");
   	scanf("%f",&nf);
   	printf("Nota do exame: %.1f\n",nf);
   	if ((nf+media1)/2>=5) printf ("Aluno aprovado.\n");
   	if ((nf+media1)/2<5) printf ("Aluno reprovado.\n");
   	printf ("Media final: %.1f\n",(nf+media1)/2);
   	
   	return 0;
   	
   	
   }
 
 
 
 
 
 return 0;
 		
   
}

File: Python/1040.py
a, b, c, d = input().split()
a = float(a)
b = float(b)
c = float(c)
d = float(d)

media = ((a*2) + (b*3) + (c*4) + d)/10

print(f'Media: {media:.1f}')
if media >= 7.0:
    print(f'Aluno aprovado.')
elif media >= 5.0:
    print(f'Aluno em exame.')
    notaF = float(input())
    mediaF = (media + notaF)/2.0
    print(f'Nota do exame: {notaF:.1f}')
    if mediaF >= 5.0:
        print(f'Aluno aprovado.')
    else:
        print(f'Aluno reprovado.')
    print(f'Media final: {mediaF:.1f}')
else:
    print(f'Aluno reprovado.')